nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss

```diff
@@ -354,8 +353,9 @@
     }
 
     .property-editor {
-        background-color: $nifi-theme-surface-palette-darker;
-        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.25);
+        background-color: var(--mat-menu-container-color);
+        box-shadow: 0 3px 6px rgba(0, 0, 0, 0.25);
+        border-radius: 4px;
```

Comment:
The resize handle is rendered on top of the bottom right corner. We may want to consider updates there now that the modal uses a larger border radius.

![Screenshot 2024-04-30 at 8 58 54 AM](https://github.com/apache/nifi/assets/123395/4c8fd1a2-c0e7-415a-8bc6-5b33f443a97d)

Reply:
Good catch. I have updated it to be positioned like the other resized handles:
![Screenshot 2024-04-30 at 12 37 55 PM](https://github.com/apache/nifi/assets/6797571/78563995-a128-4d3d-9b88-3906141fd79f)